server/lib/resolve/index.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
 exports.id = require('./id');
+exports.canAllGraphicsBeSyndicated = require('./canAllGraphicsBeSyndicated');
 exports.canDownload = require('./canDownload');
 exports.canBeSyndicated = require('./canBeSyndicated');
 exports.downloaded = require('./downloaded');
 exports.embargoPeriod = require('./embargoPeriod');
+exports.hasGraphics = require('./hasGraphics');
```

Comment:
Are you sure that these files are necessary? Considering that you then remove the properties [here](https://github.com/Financial-Times/next-syndication-api/pull/235/files#diff-e2babe4577db9910d30c6c07532ca665R73) and that what you your code execute during the enrichment and the enrichment seems to specific only for articles.

Reply:
Looks like it is necessary. The resolve controller passes an empty object to `item` and the enriched content to `src`.
https://github.com/Financial-Times/next-syndication-api/blob/master/server/controllers/resolve.js#L47-L53

If the property isn't included in the folder, it doesn't get set in the item object.
https://github.com/Financial-Times/next-syndication-api/blob/master/server/lib/syndicate-content.js#L40

Reply:
Yeah, it's really odd way to do it, but I'm guessing whoever built it has a reason to do so?

Reply:
They do that cause the same function is [used here](https://github.com/Financial-Times/next-syndication-api/blob/42bb5ac58119c40b34a1598632debb2909f8d608/server/controllers/translations.js#L132) but passing the original item to prepare the syndicate object for the Spanish content.

Let's keep in mind about the ES part. Do we need to do anything about that?